resources/function_help/json/color_mix_rgb

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "color_mix_rgb",
+  "type": "function",
+  "description": "Returns a string representation color mixing two provided colors based on a given ratio.",
```

Comment:
Maybe explicility state here that this is done by mixing the r/g/b values separately?